src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -11,8 +12,10 @@
 import com.minecolonies.tileentities.TileEntityColonyBuilding;
 import com.minecolonies.util.BlockPosUtil;
 import com.minecolonies.util.Log;
+import com.minecolonies.util.ServerUtils;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.util.ServerUtils'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)